### DISCUSSION TITLE
Just a question - please ignore and delete if breaking any rules

### DISCUSSION THREAD

**Discussion by MMarius84** [ChefKissInc/NootRX#88] [General]
### macOS Version

Sonoma

### What is your CPU's model?

Intel i5-6500

### What is your GPU's model?

AMD RX6400

### Please describe the feature in detail.

Hi there,
I am running macOS on a Dell 5040 SFF which limits the options that I have for a dedicated GPU (due to size and power constraints) which led me to get an RX550 (I know it's not NAVI, it's Lexa and needs spoofing to Baffin) but was looking at the RX6400 (Navi24) as it fits in both in terms of size and power (and price :-) ). 
I have seen (and have been following for quite a while now the [already opened ticket #45](https://github.com/ChefKissInc/NootRX/issues/45) and just wanted to ask if there is any work being made in that direction. Would it make sense in keeping my hopes up in getting that or should I look into getting a completely different system that could fit a bigger GPU? I do not mind holding off and waiting for a few more months or so.

With the highest respect for all the work you have been doing, I do not wish to cross any boundaries or upset anyone, just asking if there's any info or updates that could be shared if that is within reason.

I thank you in advance and wish you a beautiful day.
Thank. 

### What alternatives have you tried, if any?

Just a question at this point

**Comment by VisualEhrmanntraut** [comment]
I am making some progress slowly, yes.

**Reply by MMarius84** [reply]
You are the beast. I swear to God you just made my day :-D
I would love to have your knowledge and skills or help in any way.

Thank you so much for all the work you have done and for the quick reply, you are a legend. 👍

**Reply by VisualEhrmanntraut** [reply]
You're welcome. Semi-unrelated but I am thinking about merging NootRX into NootedRed so you might see this repo get archived at some point.

**Reply by VisualEhrmanntraut** [reply]
@MMarius84 I think I'll need more testers, can you contact me on Telegram? (@VisualEhrmanntraut just like my GitHub)

**Reply by MMarius84** [reply]
Hey, thanks a lot for the info, will be keeping an eye out for changes on the repo.
In regards to testing, sure thing, would love to help the project and the community.
Don't have telegram, but will get it and look you up.

**Reply by N-R-G** [reply]
Hi @VisualEhrmanntraut 
I just bought a radeon navi 24....  Radeon rx 6500 xt 8GB...I hope your project about it will go ahead.... Thank you for your fantastic work
